Source/Initialization/DivCleaner/ProjectionDivCleaner.cpp

```
@@ -151,7 +152,7 @@ ProjectionDivCleaner::solve ()
         auto ithi = bcmap.find(WarpX::field_boundary_hi[idim]);
         if (itlo == bcmap.end() || ithi == bcmap.end()) {
             WARPX_ABORT_WITH_MESSAGE(
-                "Field boundary conditions have to be either periodic, PEC or neumann "
+                "Field boundary conditions have to be either periodic, PEC, PMC, or neumann "
```

Turns out Neumann does nothing for electromagnetic. Should we make it such that the BC defaults to PMC of if the user specifies Neumann for EM simulation?

---

I would say that Neumann should return an error with electromagnetic. I was thinking that that change would be a separate PR, but it could be added to this one.

---

I think it could make sense to do in this PR. 

We should change the neumann BC to pmc in the pec_insulator CI tests, which are what motivated this PR in the first place.

---

I thought about this some more and changed my mind. Since Neumann is the same thing as PMC, Neumann should be allowed with EM. The question is do we allow both Neumann and PMC as input? It looks like we can have the two different names but with the same value so either could be used. I'll make the changes.